packages/java/tests/spring/endpoints/src/main/java/com/vaadin/flow/connect/ObjectWithNullValues.java

```diff
@@ -2,4 +2,8 @@
 
 public class ObjectWithNullValues {
     private String propWithNullValue;
+
+    public String getPropWithNullValue() {
+        return propWithNullValue;
+    }
```

Comment:
I'd try to avoid modifying existing tests and use a configured `ObjectMapper` instead. That would also allow to offer backwards compatibility as an option.